contracts/test/Imports.sol

```diff
@@ -5,9 +5,9 @@ import "@aragon/os/contracts/kernel/Kernel.sol";
 import "@aragon/os/contracts/factory/DAOFactory.sol";
 import "@aragon/os/contracts/factory/EVMScriptRegistryFactory.sol";
 
-import "@aragon/staking/contracts/StakingFactory.sol";
 import "@aragon/apps-agreement/contracts/test/mocks/AgreementMock.sol";
 import "@aragon/apps-agreement/contracts/test/mocks/disputable/ArbitratorMock.sol";
+import "@aragon/apps-agreement/contracts/test/mocks/disputable/AragonAppFeesCashierMock.sol";
```

Comment:
I'm having issues when compiling the contracts as `@aragon/apps-agreement` doesn't seem to have the `test` mock contracts available anymore.

Reply:
You need to checkout the latest version of agreements and link it.

Reply:
I've brought this up with them so hopefully they'll include the test mock contracts in the next package update.